plugins/web/opentelemetry-instrumentation-page-view/src/instrumentation.ts

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { InstrumentationBase, isWrapped } from '@opentelemetry/instrumentation';
+import { LogRecord, logs, Logger } from '@opentelemetry/api-logs';
+import { PACKAGE_NAME, PACKAGE_VERSION } from './version';
+import {
+  ApplyCustomLogRecordDataFunction,
+  PageViewInstrumentationConfig,
+} from './types';
+import { PageTypes } from './enums/PageTypes';
+/**
+ * This class represents a page view instrumentation plugin
+ */
+const EVENT_NAME = 'browser.page_view';
+export class PageViewInstrumentation extends InstrumentationBase<PageViewInstrumentationConfig> {
+  eventLogger: Logger | null = null;
+  oldUrl = location.href;
+  applyCustomLogRecordData: ApplyCustomLogRecordDataFunction | undefined =
+    undefined;
+
+  /**
+   *
+   * @param config
+   */
+  constructor(config: PageViewInstrumentationConfig) {
+    super(PACKAGE_NAME, PACKAGE_VERSION, config);
+    this.eventLogger = logs.getLogger(PACKAGE_NAME, PACKAGE_VERSION);
+    this.applyCustomLogRecordData = config?.applyCustomLogRecordData;
+  }
+
+  init() {}
+
+  /**
+   * callback to be executed when using hard navigation
+   */
+  private _onPageView() {
+    const pageViewLogRecord: LogRecord = {
+      attributes: {
+        'event.name': EVENT_NAME,
```

Comment:
We should use the `eventName` field instead.

Reply:
I don't see evetName in LogRecord